cmd/swctl/app/cmd_dependency.go

```diff
@@ -491,6 +491,14 @@ func InstallDocker(cli Cli, dockerVersion string) error {
 		color.Fprintln(cli.Out(), out)
 
 	}
+	errorGroup := postInstall(cli)
```

Comment:
Please, rename to `dockerPostInstall` as it is not sure to which install is this post install.